telegram/_telegramobject.py

```diff
@@ -312,7 +313,18 @@ def __setstate__(self, state: Dict[str, object]) -> None:
             try:
                 setattr(self, key, val)
             except AttributeError:
-                # catch cases when old attributes are removed from new versions
+                # So an attribute was deprecated and removed from the class. Let's handle this:
+                # 1) Is the attribute now a property with no setter? Let's check that:
+                if isinstance(getattr(self.__class__, key, None), property):
+                    # It is, so let's try to set the "private attribute" instead
+                    try:
+                        setattr(self, f"_{key}", val)
+                    except AttributeError:  # guess we've completely removed it
+                        api_kwargs[key] = val  # add it to api_kwargs as fallback
```

Comment:
very nitpicky: you're mixing inline comments with block comments (block comments are prefered by pep8 and me 🤡 )